expconfigs/exp.config-0.25

```diff
@@ -0,0 +1,117 @@
+DA_VARIABLES_ICE='hsnon, hicen, cicen'
+DA_VARIABLES="hsnon, hicen, cicen, socn, tocn, ssh, hocn"
+DA_CHKPT_WITH_MODEL=T
+DA_DIAGB_ENABLED=F
+DA_DIAGB_DIRAC_STEP=40
+DA_DUALRES_ENABLE=F
+DA_OCN_IAU=T
+DA_DART=F
+
+#-------------------------------------------------------------------------------
+# model configuration
+#-------------------------------------------------------------------------------
+MODEL=ufs
+UFS_SRC_DIR=/work/noaa/ng-godas/ycteng/runs/sandbox/20211112/ufs_config
+MODEL_EXE=$UFS_SRC_DIR/fv3_20211101_IAU.exe
+#UFS_SRC_DIR=/work/noaa/marine/Guillaume.Vernieres/runs/s2s/adt/adt.sigo.reperr.iau/ufs_config
+#MODEL_EXE=/work/noaa/marine/Guillaume.Vernieres/runs/s2s/ufs-weather-model/build/ufs_model
+MODEL_RES=0.25deg
+MODEL_CFG_DIR=$UFS_SRC_DIR/CFG_DIR/$MODEL_RES
+MODEL_DATA_DIR=$UFS_SRC_DIR/INPUT/$MODEL_RES
+
+# UFS specific variables
+UFS_SCRATCH="$UFS_SRC_DIR/SCRATCH/$MODEL_RES"
+UFS_INSTALL="$UFS_SRC_DIR/INSTALL"
+UFS_ATMPETS=72
+UFS_OCNPETS=360
+UFS_ICEPETS=48
+UFS_MEDPETS=72 #96
+DT_CICE=900
+
+#-------------------------------------------------------------------------------
+# initial conditions (background restart files)
+#-------------------------------------------------------------------------------
+BKGRST_SRC="/work/noaa/da/ycteng/sandbox/example/ng_RT/initial"
+BKGRST_COLDSTART_ENABLED=F
+
+
+#-------------------------------------------------------------------------------
+# Forcing
+#-------------------------------------------------------------------------------
+FORC_ENABLED=F                   # If =T, forcing files are required by the model
+FORC_SRC="/work/noaa/ng-godas/marineda/DATM_INPUT"  # Forcing files will be obtained from here if possible
+FORC_GEN_ENABLED=F               # If forcing files can't be found, regenerate them
+FORC_GEN_SOURCE=CFSR             # only gfs is available currently
+CFSR_BIAS_CORR=T
+DATM_START_TYPE="startup"
+ICE_START_TYPE="initial"
+DATM_NAME="cdeps"
+
+
+#-------------------------------------------------------------------------------
+# Observations
+#-------------------------------------------------------------------------------
+R2D2_ENABLED=T
+R2D2_DB_DIR=/work/noaa/ng-godas/r2d2/
+R2D2_PROVIDER="gdas_marine"
+R2D2_EXP="s2s_v1"
+
+OBS_ENABLED=T
+OBS_TOLERATE_FAIL=T
+OBS_GEN_ENABLED=F
+
+OBS_LIST_OCN="temp salt adt sst sss icec"
+              
+OBS_ADT_LIST="3a_egm2008 3b_egm2008 6a_egm2008 j2_egm2008 j3_egm2008 c2_egm2008 sa_egm2008 coperl4"
+OBS_SST_LIST="drifter ship_fnmoc trak_fnmoc metopa_l3u_so025 metopb_l3u_so025 metopc_l3u_so025 noaa18_l3u_so025 noaa19_l3u_so025 noaa20_l3u_so025 viirs_npp_l3u_so025 viirs_n20_l3u_so025"
+OBS_SSS_LIST="trak_fnmoc"
+OBS_SALT_LIST="profile_fnmoc"
+OBS_TEMP_LIST="profile_fnmoc"
+OBS_ICEC_LIST="ssmis_f17_north ssmis_f17_south ssmis_f18_north ssmis_f18_south"
+
+OBS_DB_PT10M="sst_metopa_l3u_so025 sst_metopb_l3u_so025 sst_metopc_l3u_so025 sst_noaa18_l3u_so025 sst_noaa19_l3u_so025 sst_noaa20_l3u_so025 sst_viirs_npp_l3u_so025 sst_viirs_n20_l3u_so025"
```

Comment:
Why do we have sst_noaa20_l3u_so025?

Reply:
Thanks for pointing that out @ShastriPaturi . I removed mentions of noaa20 in both configs.